packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS sentSubscriptionEmails (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  emailType VARCHAR(60) NOT NULL,
```

Comment:
Templates don't really change often but we do associate a version number with them if that info might be needed.

Reply:
Thanks for the reminder about template versions.  I don't think we'll need it here since we mainly want to know what type of email was sent not the content.

> Templates don't really change often

I struggled a tiny bit with whether to normalize this more with an email types table.  It seems a little fragile requiring devs to remember manually add to the email types table whenever they add more email templates, all the while we don't really gain much from it.

Do let me know if you feel strongly about this one way or the other.

Reply:
> Do let me know if you feel strongly about this one way or the other.

I don't feel strongly about this. The table would be fairly small compared to other tables.

Reply:
> 
> 
> > Do let me know if you feel strongly about this one way or the other.
> 
> I don't feel strongly about this. The table would be fairly small compared to other tables.

Because @bbangert suggested that the table should be made generic to be used for _all_ emails, and since the number of FxA users is orders of magnitude greater than the number of SubPlat subscribers, the potential size and growth of the table just got much bigger.  I'll add a table for the email types in my updated patch.  Thanks for the input!

Reply:
Since there hasn't been any strong interest in having this to track all emails sent, I'm good with this as-is.